syntax = "proto3";

package rj_OfflineServer;

message Request
{
    bytes type = 1;
    bytes request_msg = 2;
}

message WriteOfflineRequest
{
    int32 user_id = 1;
    int32 friend_id = 2;
    bytes msg = 3;
}

message ReadOfflineRequest
{
    int32 myid = 1;
}

message from_message
{
    int32 from_id = 1;
    bytes msg = 2;
}

message ReadOfflineResponse
{
    bool is_success= 1;
    repeated from_message msgs= 2;
}
